include/avt_vimba_camera/mono_camera.h

```diff
@@ -43,19 +43,22 @@
 #include <camera_info_manager/camera_info_manager.h>
 #include <image_transport/image_transport.h>
 #include <dynamic_reconfigure/server.h>
-
+#include <cav_msgs/SystemAlert.h>
+#include <cav_msgs/DriverStatus.h>
 #include <string>
 
 namespace avt_vimba_camera {
 class MonoCamera {
  public:
   MonoCamera(ros::NodeHandle& nh, ros::NodeHandle& nhp);
   ~MonoCamera(void);
-
+ void time_compare();
+ int flag;
```

Comment:
Variable flag is used to track the internal status of the camera. A better name and a small data type should be used. Also it is a good habit to avoid public variables and use getter and setter methods.